frontend/src/components/JoinUs.vue

```diff
@@ -15,8 +15,8 @@
           <p class="content">
             Kodujesz? Jesteś grafikiem? UX-owcem? Robisz cokolwiek innego, co
             może się przydać w kodowaniu dla Poznania? Dołącz do nas! Spotykamy
-            się w każdą środę od godz. 17:00 w firmie STX Next przy ul.
-            Morawskiego 12 (koło Areny).
+            się w każdą środę od godz. 18:00 w firmie STX Next przy ul. Mostowej
+            38 niedaleko Pawilonu w parku Stare Koryto Warty.
```

Comment:
Nie wiem, czy Nowa Gazownia nie będzie lepszą nazwą. Lub też Nowa Gazownia (Pawilon).

Reply:
No ja właśnie nie wiedziałem jak to się nazywa ^^ @agata-m co myślisz?